frontend/src/pages/GraphPF/GraphPFElems.tsx

```diff
@@ -254,6 +255,11 @@ export const setNodeLabel = (node: NodeModel, nodeMap: NodeMap, settings: GraphP
     if (data.isGateway?.egressInfo?.hostnames?.length !== undefined) {
       data.labelIcon = <span className={`${badgeMap.get('GW')?.className}`}></span>;
     }
+    if (data.isWaypoint) {
+      data.labelIcon = (
+        <span className={`${badgeMap.get('WA')?.className}`} style={{ marginBottom: '1px', marginLeft: '2px' }}></span>
+      );
+    }
```

Comment:
This is OK because we being a Waypoint should be mutually exclusive with being a traffic source. But we may want to update the comment above (in a different PR) to reflect this case.